package.json

```diff
@@ -48,26 +46,27 @@
     "rimraf": "^3.0.2"
   },
   "peerDependencies": {
-    "expo": "*",
+    "expo": ">=47.0.0",
```

Comment:
this is why I think this is a breaking change. In theory, future SDK updates won't, tho

Reply:
Yeah, we probably don't want to use the re-exported ones for that reason, I don't think we should make it a breaking release just for that.

Reply:
if #301 is breaking, then they can both go in?